testing/dbtest/docker/postgresql.conf

```diff
@@ -0,0 +1,32 @@
+listen_addresses           = '*'
```

Comment:
I have no insight or expertise to comment on these settings overall.

Reply:
Do you think this needs some comments to explain the options? Or is it ok to just point people to the official postgres docs if they want to learn about each setting?

Reply:
It might be helpful to give a very succinct reason for each setting so devs understand why and then go explore the docs if they want more info about each setting.

Reply:
Added some comments to the config